plane/src/typed_unix_socket/mod.rs

```diff
@@ -38,25 +25,25 @@ fn get_quick_backoff() -> ExponentialBackoff {
 mod tests {
     use super::client::TypedUnixSocketClient;
     use super::server::TypedUnixSocketServer;
+    use crate::util::random_string;
     use futures_util::future::join_all;
     use std::collections::HashSet;
     use std::env;
     use std::path::PathBuf;
     use tokio::spawn;
-    use uuid::Uuid;
 
     fn create_temp_socket_path() -> PathBuf {
-        env::temp_dir().join(format!("test_socket_{}", Uuid::new_v4())) // ensure random file name
+        env::temp_dir().join(format!("test_socket_{}", random_string())) // ensure random file name
```

Comment:
`random_string()` over `random_token()` is important here because a socket path can only be max 108 bytes ([ref](https://man7.org/linux/man-pages/man7/unix.7.html)) and it is borderline with `random_token()` which is longer.

Fortunately, there is absolutely zero chance of collision here because paths are already scoped by test.

Reply:
Nit: seems like a good idea to put something like this in a comment.